server/server.js

```diff
@@ -112,6 +114,17 @@ module.exports = async ({graphQLConfig, graphiqlConfig, postgresConfig, schemaLi
   process.on('SIGQUIT', stopHandler)
   process.on('SIGINT', stopHandler)
 
+  // Set up the WebSocket for handling GraphQL subscriptions
+  /* eslint-disable no-new */
+  new SubscriptionServer({
```

Comment:
this won't pick up new schema with hot reload

Reply:
👍 This completely slipped my mind